.eslintrc

```diff
@@ -25,14 +25,17 @@
   },
   "overrides": [
     {
-      "files": ["*.test.js"],
-      "env": {
-        "mocha": true
+      "files": ["*.ts", "*.tsx"],
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "prettier"
+      ],
+      "parser": "@typescript-eslint/parser",
+      "plugins": ["@typescript-eslint"],
+      "rules": {
+        "@typescript-eslint/no-explicit-any": "off"
       }
-    },
-    {
-      "files": ["**/*.html"],
-      "plugins": ["eslint-plugin-html"]
     }
```

Comment:
You still need this one in there. You are supposed to _add_ the `*.ts, *.tsx` override to the list of "overrides", not replace it.